README.md

```diff
@@ -1,3 +1,42 @@
 # TheSportsDB
 GraphQL wrapper for TheSportsDB
 API doc - https://www.thesportsdb.com/api.php
+
+# Example of query
+
+Query for get types of schema
```

Comment:
Hi @ufoo68 ,
Could you please elaborate on why this documentation is needed in the README?
The beauty of GraphQL is that you don't have to document the API, it documents itself with the schema and types - all you have to do is open up the playground and everything is already documented and available.

I see value, however, in adding documentation on server execution details and how to access the playground. Would you like to edit the PR to include this information?

Reply:
I understand. I will change the document describe how to execute server and access playground.

Reply:
[f89e35c](https://github.com/jsBlackBelt/TheSportsDB/pull/2/commits/f89e35c1028c3d8ca7985378c969a3f7430cd26e)